src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,111 @@
+package duke;
+
+/**
+ * The Parser class that makes sense of the user input.
+ */
+public class Parser {
+    private String input;
+    private String[] inputArr;
+    private String command;
+
+    /**
+     * Constructor for a Parser object.
+     *
+     * @param input The user input to be parsed.
+     */
+    public Parser(String input) {
+        this.input = input;
+        this.inputArr = input.split(" ");
+        this.command = inputArr[0];
+    }
+
+    private boolean isEditingTask() {
+        return command.equals("done") || command.equals("delete");
+    }
+
+    private boolean isFindingTask() {
+        return command.equals("find");
+    }
+
+    private boolean isAddingNewTask() {
+        return command.equals("todo") || command.equals("deadline") || command.equals("event");
```

Comment:
Perhaps this should be done is steps. For example
```
boolean isTodo = command.equals("todo");
boolean isDeadline = command.equals("deadline");
boolean isEvent = command.equals("event");
return isTodo || isDeadline || isEvent;
```

Reply:
Thanks for the suggestion, will make the change 👍